bpaf_core/src/visitor/explain_unparsed.rs

```diff
@@ -127,6 +130,25 @@ impl<'a> ExplainUnparsed<'a> {
         }
         None
     }
+    fn is_redundant(&self, parsed: &[Name], unparsed: Name) -> Option<Error> {
+        let unparsed_info = self.all_names.get(&unparsed)?;
+        if unparsed_info.in_many {
```

Comment:
This check makes sense. If parser is annotated with `many` in any way - it can be accepted several times so this diagnostic is no longer applies.